s/frontend/components/prepare-components.ts

```diff
@@ -11,6 +15,14 @@ export function prepareComponents({models: {commenting}}: {
 
 	return themeComponents(themeCss, {
 
+		MyReview,
```

Comment:
prefix all components with `Namaka`, so they are

`<namaka-my-review>`

this simple namespace prevents us from ever colliding with other web component libraries

Reply:
Done